src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/perform_interaction_request.h

```diff
@@ -93,6 +101,32 @@ class PerformInteractionRequest
    */
   virtual void onTimeOut();
 
+  /**
+   * @brief Prepare result code for sending to mobile application
+   * @param first contains result_code from HMI response and
+   * interface that returns response
+   * @param second contains result_code from HMI response and
+   * interface that returns response.
+   * @return resulting code for sending to mobile application.
+   */
+  mobile_apis::Result::eType PrepareResultCodeForResponse(
```

Comment:
@YarikMamykin please, consider moving this and the next method to the protected or private section.

Reply:
@mkorniichuk please see a2ef4e9